src/dune_file_watcher/dune_file_watcher.ml

```diff
@@ -590,11 +684,12 @@ let create_default ?fsevents_debounce ~scheduler () =
       ~debounce_interval:(Some 0.5 (* seconds *)) ~backend
   | `Fsevents -> create_fsevents ?latency:fsevents_debounce ~scheduler ()
   | `Inotify_lib -> create_inotifylib ~scheduler
+  | `Winwatch -> create_winwatch ~scheduler ~debounce_interval:0.5
```

Comment:
let's group this `debounce_interval` constant with the fswatch one in a module global.